code/modules/busy_space/systems.dm

```diff
@@ -0,0 +1,602 @@
+		new /datum/lore/location("Seledon on Titan, Saturn subsystem, Sol", list(ATC_TYPICAL)),
+		new /datum/lore/location("Hammar on Titan, Saturn subsystem, Sol", list(ATC_TYPICAL)),
+		new /datum/lore/location("Bolsena on Titan, Saturn subsystem, Sol", list(ATC_TYPICAL)),
+		new /datum/lore/location("Sotra on Titan, Saturn subsystem, Sol", list(ATC_TYPICAL)),
+		new /datum/lore/location("Mercury, Sol", list(ATC_TYPICAL)),
+		new /datum/lore/location("Callisto, Jupiter subsystem, Sol", list(ATC_TYPICAL)),
+		new /datum/lore/location("Ganymede, Jupiter subsystem, Sol", list(ATC_TYPICAL)),
+		new /datum/lore/location("Io, Jupiter subsystem, Sol", list(ATC_TYPICAL)),
+		new /datum/lore/location("a detention facility on Io, Jupiter subsystem, Sol", list(ATC_MED, ATC_TRANS)),
+		new /datum/lore/location("Europa, Jupiter subsystem, Sol", list(ATC_ALL_CIV)),
+		new /datum/lore/location("Titania, Uranus subsystem, Sol", list(ATC_TYPICAL)),
+		new /datum/lore/location("Oberon, Uranus subsystem, Sol", list(ATC_TYPICAL, ATC_SCI)), //there's ancient discord lore that has a school here??
+		new /datum/lore/location("Neptune, Sol", list(ATC_TYPICAL)),
+		)
+
+/datum/lore/system/vir
+	name = "Vir"
+	desc = "You Are Here. A crossroads system with a tendency towards sleepy isolation."
+	autogenerate_destinations = FALSE
+	planets = list("Sif")
+	locations = list(
+		new /datum/lore/location("New Rekjavik", list(ATC_ALL_CIV)), //sivian locations don't need 'on sif' and virite locations dont need 'in vir'-- theyre local
+		new /datum/lore/location("Kalmar", list(ATC_ALL_CIV)),
+		new /datum/lore/location("Ekmanshalvo", list(ATC_ALL_CIV)),
+		new /datum/lore/location("a settlement on the Thorvaldsson Plains", list(ATC_TYPICAL)), //padding
+		new /datum/lore/location("a mining outpost in the Dauthabrekka Mountains", list(ATC_TYPICAL)),
+		new /datum/lore/location("a corporate facility in Stockholm-Skargard", list(ATC_TYPICAL, ATC_SCI)),
+		new /datum/lore/location("a site in the Ullran Expanse", list(ATC_MED, ATC_DEF, ATC_INDU)),
+		new /datum/lore/location("a location in the Anomalous Region", list(ATC_SCI)),
+		new /datum/lore/location("the NLS Southern Cross", list(ATC_DEF, ATC_INDU)),
+		new /datum/lore/location("the NCS Northern Star", list(ATC_TYPICAL, ATC_SCI)),
+		new /datum/lore/location("Vir Interstellar Spaceport", list(ATC_TYPICAL)),
+		new /datum/lore/location("Firnir", list(ATC_TYPICAL)),
+		new /datum/lore/location("Tyr", list(ATC_TYPICAL)),
+		new /datum/lore/location("Magnai", list(ATC_TYPICAL)),
+		new /datum/lore/location("the rings of Kara", list(ATC_LUX)),
+		new /datum/lore/location("the rings of Rota", list(ATC_LUX)),
+		new /datum/lore/location("the Radiance Energy Chain", list(ATC_FREIGHT, ATC_DEF, ATC_INDU)),
+		new /datum/lore/location("a Nanotrasen colony in the Kara subsystem", list(ATC_ALL_CIV)), //padding-- there should probably be more than one named kara colony anyway
```

Comment:
I think the NS is really THE "company town". Maybe "NanoTrasen facility" more generally, for other work sites, mines, labs etc.?